recipe/meta.yaml

```diff
@@ -21,8 +21,9 @@ requirements:
     - python >=3.7
     - setuptools-scm
   run:
-    - botorch ==0.4.0
+    - botorch ==0.6.2
     - jinja2
+    - multipledispatch
```

Comment:
```suggestion
    - multipledispatch
    - sqlalchemy
```

Error says
>     ModuleNotFoundError: No module named 'sqlalchemy'

Reply:
https://github.com/facebook/Ax/blob/main/setup.py#L47

Reply:
I guess it's gonna say it needs jupyter as well, probably

Reply:
that the thing, it is quite modular, but you are testing against many of the submodules. So it is your choice if you want this to be a "full package" or a bare minimum. People tend to oscillate

Reply:
yeah, I'm not sure either, but I guess it's not that huge of a deal to include a couple more pacakges